tagged/tagged.go

```diff
@@ -0,0 +1,131 @@
+package tagged
+
+import (
+	"fmt"
+	"reflect"
+)
+
+// Value holds an arbitrary value with associated tags
+type Value[T comparable] struct {
```

Comment:
I feel this should be broken out into a repo separate from stereoscope if we're going to use it in a more generic (ha) sense